Sources/WebService/AuthenticatedWebService/AuthenticatedWebService.swift

```diff
@@ -75,7 +75,7 @@ open class AuthenticatedWebService: WebService {
           return appendTokenAndExecute(accessToken: accessToken)
       }
     }
-    .timeout(10, scheduler: DispatchQueue.main, customError: { NetworkError.timeout })
+    .timeout(.init(floatLiteral: urlRequest.timeoutInterval), scheduler: DispatchQueue.main, customError: { NetworkError.timeout })
```

Comment:
I think this is not needed as url session will handle it automatically under the hood, so you'll `receive URLSessionError` instead of failing the whole stream

Reply:
Are you sure this will timeout under all circumstances? What if `self.tokenProvider.accessToken` in line 71 does not publish a token?

Reply:
then it's different issue, no?

Reply:
but what if I want to abort the stream after a specified time?

Reply:
then you should do it outside of the framework, no?

Reply:
For me it looks like we are mixing **stream-based timeouts** with **network session timeouts**.

A `URLSession` can be configured by a [URLSessionConfiguration](https://developer.apple.com/documentation/foundation/urlsessionconfiguration) where you can set the timeout.

The question remains if the `URLSession` gets the `timeoutInterval` from an `URLRequest` and automatically updates the `URLSessionConfiguration`. That is something we need to test and cover in a networking framework rather than having arbitrary constants in place.

Reply:
> The question remains if the `URLSession` gets the `timeoutInterval` from an `URLRequest` and automatically updates the `URLSessionConfiguration`. That is something we need to test and cover in a networking framework rather than having arbitrary constants in place.

Yes the `URLSession` gets the `timeoutInterval` from an `URLRequest`, this works fine and as expected. I wanted to have a possibility to abort the stream after a specified time, but as @GlebRadchenko suggested this is also doable outside the framework. I will change this PR accordingly.

Reply:
I pushed my changes and updated the title and description of the PR. @GlebRadchenko kindly review again.